core/sis-referencing/src/test/java/org/apache/sis/geometry/CoordinateFormatTest.java

```diff
@@ -188,4 +193,76 @@ public void testParseGeographic4D() throws ParseException {
                          e.getLocalizedMessage());  // In the language specified at CoordinateFormat construction time.
         }
     }
+
+    @Test
+    public void testCreatesCoordinateFormatTaking2ArgumentsAndCallsClone() {
+        Locale locale = Locale.CANADA;
+        CoordinateFormat coordinateFormat = new CoordinateFormat(locale, null);
+        CoordinateFormat coordinateFormatTwo = coordinateFormat.clone();
+
+        assertNotSame(coordinateFormatTwo, coordinateFormat);
+    }
+
+    @Test
+    public void testParseAndFormatTaking2Arguments() throws IOException, ParseException {
+        CoordinateFormat coordinateFormat = new CoordinateFormat();
+        DirectPosition2D directPosition2D = new DirectPosition2D();
+        StringBuffer stringBuffer = new StringBuffer();
+        coordinateFormat.format(directPosition2D, stringBuffer);
+        ParsePosition parsePosition = new ParsePosition(0);
+        DirectPosition directPosition = coordinateFormat.parse(stringBuffer, parsePosition);
+
+        assertEquals(2, directPosition.getDimension());
+        assertEquals(2, directPosition.getCoordinate().length);
+        assertEquals(0.0, directPosition.getCoordinate()[0], 0.01);
```

Comment:
We expect exact result here since the coordinate value is not the result of a calculation. So the tolerance threshold can be set to 0 instead of 0.01. I will make that chance on my locale copy I'm reviewing.